vm/src/builtins.rs

```diff
@@ -735,7 +735,13 @@ fn builtin_zip(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
 
 // builtin___import__
 
-pub fn make_module(ctx: &PyContext) -> PyObjectRef {
+/// Build the builtin module,
+///
+/// * ctx: The Python Context
+/// * builtin_overrides: A HashMap that contains (builtin item name, PyObjectRef to that item),
```

Comment:
Instead of overrides, try to use the `set_attr` method to update a builtin function.